gating/check/run_deploy_mnaio.sh

```diff
@@ -128,7 +128,12 @@ source /opt/rpc-openstack/RE_ENV
 # RO-4206
 # Use fork of Ansible which exposes the apt errors so that we
 # can diagnose the cause of the apt fetch failures.
-export ANSIBLE_PACKAGE="git+https://github.com/rcbops/ansible@v2.4.4.0-with_apt_errors"
+# export ANSIBLE_PACKAGE="git+https://github.com/rcbops/ansible@v2.4.4.0-with_apt_errors"
+
+# RO-4356
+# Until we can update our fork of Ansible we will need to use the
+# acceptable upstream version.
+export ANSIBLE_PACKAGE="git+https://github.com/ansible/ansible@v2.5.5"
```

Comment:
If we are using the upstream version, does upstream export this for us (so we don't have to export here)?

Reply:
Just remove it and use the upstream version. Ansible 2.5 onwards has the extra diagnostic information for the apt failures built-in.

Reply:
Will do, thanks for the feedback